packages/runtime-core/src/vgo_runtime/powershell_bridge.py

```
@@ -0,0 +1,128 @@
+        raise RuntimeError(f"{bridge_label} returned invalid JSON stdout{detail}") from last_json_error
+    raise RuntimeError(f"{bridge_label} returned undecodable JSON stdout{detail}")
+
+
+def run_powershell_json_command(
+    command: Sequence[str],
+    *,
+    cwd: Path,
+    bridge_label: str,
+    env: Mapping[str, str] | None = None,
+) -> dict[str, Any]:
+    completed = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)